abstract.js

```diff
@@ -1,6 +1,8 @@
 const { Readable } = require('stream')
 const Packet = require('aedes-packet')
 
+const HIGH_WATERMARK = 64
```

Comment:
could we put the constant in the local test scope?

Reply:
I removed the constant and derive it from Readble now ;-)
I also removed a potential race condition where the submission of messages could still be running while reading the result.